src/latency.c

```diff
@@ -612,11 +618,14 @@ void latencyCommandReplyWithLatestEvents(client *c) {
         struct latencyTimeSeries *ts = dictGetVal(de);
         int last = (ts->idx + LATENCY_TS_LEN - 1) % LATENCY_TS_LEN;
 
-        addReplyArrayLen(c, 4);
+        addReplyArrayLen(c, 7);
         addReplyBulkCString(c, event);
         addReplyLongLong(c, ts->samples[last].time);
         addReplyLongLong(c, ts->samples[last].latency);
         addReplyLongLong(c, ts->max);
+        addReplyLongLong(c, ts->min);
```

Comment:
I vote to remove min. It's basically the lowest value above min, I think sum and cnt is more useful.

Reply:
I guess you mean min is the lowest value above the configured `latency-monitor-threshold`.

Reply:
Will it be useful in scenarios where `latency-monitor-threshold` may be frequently adjusted? or the real latency is much larger than `latency-monitor-threshold`?

i am not sure about it actually, i am ok with removing it if you both think it is useless.

Reply:
Only Madelyn thinks it's useless. I don't have an opinion about it. :laughing:

Reply:
I'm pretty strongly opinionated about only adding fields we are sure are useful. If we aren't sure something is useful, I would prefer to defer adding it until someone brings it up or has a use case. I prefer to lean towards simplicity as much as possible. The fewer the fields exist, the less the end users have to think about.

> Will it be useful in scenarios where latency-monitor-threshold may be frequently adjusted? or the real latency is much larger than latency-monitor-threshold?

I'm not sure what the min would tell you in that scenario.